examples/playground/components/after-paint/index.tsx

```diff
@@ -12,10 +12,10 @@ const props = {
 export const buildAfterPaintComponents = {
   server: () => {
     const WithSSR = lazyAfterPaint(() => require('./with-ssr'), {
-      moduleId: './after-paint/with-ssr',
+      moduleId: './examples/playground/components/after-paint/with-ssr.tsx',
```

Comment:
These strings were wrong as manually generated (cannot add babel plugin to playground as it's written in TS